BrainPortal/app/views/tasks/_resource_usage.html.erb

```diff
@@ -0,0 +1,44 @@
+
+<% rus = ResourceUsage.where(:cbrain_task_id => @task.id).all.to_a %>
+<% if rus.present? %>
+  <fieldset>
+    <legend>Resource Usage History</legend>
+    <table class="simple">
+      <tr>
+        <th>Date</th>
+        <th>Task Status</th>
+        <th>Usage Type</th>
+        <th>Time Used</th>
+        <th>Disk Space Used</th>
+      </tr>
+      <% rus.each do |ru| %>
+        <% type = ru.is_a?(CputimeResourceUsageForCbrainTask)  ? "CPU"        :
+                  ru.is_a?(WalltimeResourceUsageForCbrainTask) ? "Walltime"   :
+                  ru.is_a?(SpaceResourceUsageForCbrainTask)    ? "Disk Space" :
+                  "Unknown"
+        %>
+        <tr>
+          <td><%= pretty_past_date ru.created_at %></td>
+          <td><%= ru.cbrain_task_status.present? ? colored_status(ru.cbrain_task_status) : "-" %></td>
+          <td><%= type %></td>
+
+          <% if type =~ /CPU|Walltime/ %>
+            <td><%= pretty_resource_usage_value(ru) %></td>
+          <% else %>
+            <td></td>
+          <% end %>
+
+          <% if type =~ /Disk/ %>
+            <td><%= pretty_resource_usage_value(ru) %></td>
+          <% else %>
+            <td></td>
+          <% end %>
+
```

Comment:
Why did we have 2 similar if/else block?

Reply:
it's for the alignment in the table. So we get:

| [cputime] | emptycell |
| emptycell | [diskspace] |

Reply:
Thanks for the explanation.